pandas/core/indexes/datetimelike.py

```diff
@@ -31,23 +31,24 @@
 _index_doc_kwargs = dict(ibase._index_doc_kwargs)
 
 
-def ea_passthrough(name):
+def ea_passthrough(array_method):
     """
     Make an alias for a method of the underlying ExtensionArray.
 
     Parameters
     ----------
-    name : str
+    array_method : method on an Array class
 
     Returns
     -------
     method
     """
+
     def method(self, *args, **kwargs):
-        return getattr(self._eadata, name)(*args, **kwargs)
+        return array_method(self._data, *args, **kwargs)
 
-    method.__name__ = name
-    # TODO: docstrings
+    method.__name__ = array_method.__name__
+    method.__doc__ = array_method.__doc__
```

Comment:
this should address @jorisvandenbossche's point that docstrings are lost in the existing implementation